### PR TITLE
Vet clinic database: query and update animals table

### PR DESCRIPTION
**Pull request by OmarMWarraich**
### In this PR, following project requirement has been completed(Screenshots appended):

- [ ] Add a column species of type string to animals table.
- [ ] Insert the given data.
- [ ] Inside a transaction update the animals table by setting the species column to unspecified. Verify, rollback and then re verify.

- [ ] Inside a transaction update the animals table by setting the species column to digimon and pokemon based on conditions. 
- [ ] Commit and Verify.

- [ ] Inside a transaction delete all the records in the animals table, then rollback. Verify that original state exists.

- [ ] Inside a transaction, delete all animals born after Jan 1st, 2022, create savepoint, 
- [ ] Update all animals' weight to be multiplied by -1. Verify. Rollback to savepoint. Update to be * by -1. Commit and re verify.
Screenshot from 2022-11-10 04-24-56
- [ ] Write queries to the given quests.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -9,4 +9,80 @@ SELECT name FROM animals WHERE neutered = true;
...
+    Update all animals' weight to be their weight multiplied by -1. 
+    Verify that change was made. Then roll back to the save point.
+    After the rollback, verify that all records have positive weight values again.
+    Commit transaction.
+*/
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO savepoint_name;
+SELECT * FROM animals;
```
Comment (KDlamini) [inline queries.sql]: After you rollback to the savepoint
*  Kindly `UPDATE` all animals' weights that are negative to be their weight multiplied by -1 as per project requirements.

Then commit the transaction

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @OmarMWarraich 🙋‍♀️,  

Great job so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Good PR title and description
✔️  Correct use of Gitflow 
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (rloterh) [reply queries.sql]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ova,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉🌟 

## Highlights 👍
- Queries properly implemented

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- You used incorrect gitflow/ GitHub flow for this project. Kindly note that the merge request for this project should flow from **from feature into development**. Kindly note this important step for subsequent projects.
- This project appears to be worked on by a sole contributor, therefore kindly update the Author section of the ReadMe accordingly and change the header title _Authors_ to _Author_. 
- Generally, the subheading text Author1 should be replaced with your full name in your ReadMe.
- MIT license link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**